cmds/core/ping/ping.go

```diff
@@ -158,6 +160,9 @@ func main() {
 		if err != nil {
 			log.Fatalf("ping failed: %v", err)
 		}
+		if *audible {
+			fmt.Print("\a")
```

Comment:
msg = "\a" + msg
and you can get rid of the fmt.Print

Reply:
I tried the suggested appending and the and pborman/ansi package. 
However log.Print seems to omit the appended \a. I'm also in favor of getting rid of the fmt.Print but don't really see other options at the moment.